enterprise/packages/translation/package.json

```diff
@@ -16,6 +16,7 @@
     "@handlebars/parser": "^2.1.0",
     "@novu/application-generic": "workspace:*",
     "@novu/ee-dal": "workspace:*",
+    "@novu/ee-auth": "workspace:*",
```

Comment:
`@novu/ee-translation` should use authentication mechanisms directly from `@novu/ee-auth` package same as `@novu/ee-billing`

`@novu/ee-translation` will always be imported only in enterprise mode, why using `UserAuthGuard`, which insides decides conditionally between `EEUserAuthGuard` and `CommunityUserAuthGuard`, when you always know it will be `EEUserAuthGuard`